pkg/apis/scheduling/v1alpha1/types.go

```diff
@@ -107,6 +107,9 @@ const (
 
 	// PodGroupFailed means at least one of `spec.minMember` pods is failed.
 	PodGroupFailed PodGroupPhase = "Failed"
+
+	// PodGroupLabel is the default label of coscheduling
+	PodGroupLabel = "pod-group.scheduling.sigs.k8s.io"
```

Comment:
Maybe reuse the literal defined in https://github.com/kubernetes-sigs/scheduler-plugins/blob/ec632c3d7e04b7b372f9a6f4338b0dbc53ef3d46/pkg/apis/scheduling/register.go#L21?

Reply:
Sg thanks.
Done